exponent_server_sdk/__init__.py

```diff
@@ -298,6 +298,14 @@ def __init__(self, host=None, api_url=None, **kwargs):
             'max_receipt_count'] if 'max_receipt_count' in kwargs else PushClient.DEFAULT_MAX_RECEIPT_COUNT
         self.timeout = kwargs['timeout'] if 'timeout' in kwargs else None
 
+        if not self.session:
```

Comment:
```suggestion
        self.session = session
        if not self.session:
```

Session will not automatically be assigned to the class